benchmarks/bm_call_simple.py

```diff
@@ -0,0 +1,139 @@
+"""Microbenchmark for function call overhead.
+
+This measures simple function calls that are not methods, do not use varargs or
+kwargs, and do not use tuple unpacking.
+
+Taken from:
+https://github.com/python/performance/blob/9b8d859/performance/benchmarks/bm_call_simple.py
+"""
+
+import weetest
+
+
+def foo(a, b, c, d):
+  # 20 calls
+  bar(a, b, c)
```

Comment:
I wouldn't make any changes here - this is something that I would either want to capture upstream, or improve in grumpy.